.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.python-version
```

Comment:
Where does this come from?

Reply:
Good question. It may come from my `nvim` setup which needs a special python version. How do go about his? Leave it in or remove from `.gitignore`

Reply:
Removed it from the `.gitignore` and added it to global `.gitignore` at `~/.config/git/ignore`